### PR TITLE
Test matrix queue fix

### PR DESCRIPTION
**Pull request by lealjd**
Full coverage for taskiq-matrix. Bug fixes, optimizations, and test adjustments.

### CODE REVIEW

File: taskiq_matrix/filters.py [hunk 1]
```diff
@@ -181,8 +187,13 @@ async def run_room_message_filter(
         else:
             d[room_id] = [event.source for event in res.chunk]
 
-    return d, res.start if MessageDirection.back else res.end
-
+    print('start=====', res.start)
+    print('end=====', res.end)
```
Comment (justin-russell) [inline taskiq_matrix/filters.py:191]: Extra print statements

File: taskiq_matrix/matrix_queue.py [hunk 2]
```diff
@@ -238,6 +238,8 @@ async def get_tasks(
             A list of tasks and acks.
         """
         next_batch = since_token or self.checkpoint.since_token
+        print('next_batch=======', next_batch)
+        print('checkpoint next batch===========', self.checkpoint.since_token)
```
Comment (justin-russell) [inline taskiq_matrix/matrix_queue.py:242]: Extra print statements

File: taskiq_matrix/matrix_queue.py [hunk 3]
```diff
@@ -256,11 +258,10 @@ async def get_tasks(
                 self.client, task_filter, timeout=timeout, since=next_batch
             )
         else:
+            print('here')
```
Comment (justin-russell) [inline taskiq_matrix/matrix_queue.py:261]: Extra print statements

File: tests/queue/test_matrix_queue.py [hunk 4]
```diff
@@ -134,6 +137,139 @@ async def test_matrix_queue_get_tasks_(test_matrix_broker):
     print('lenth of list======', len(result))
```
Comment (justin-russell) [inline tests/queue/test_matrix_queue.py:137]: Extra print statement

**Review by justin-russell — CHANGES_REQUESTED** [review]

**Comment by justin-russell** [comment]
Also, go ahead and bump up the version in `pyproject.toml` as well!

**Comment by lealjd** [comment]
Got it all fixed up!

**Review by justin-russell — APPROVED** [review]

**Pull request merged**